addon/components/kinetic-form.js

```diff
@@ -50,7 +50,15 @@ export default Component.extend({
       let validators = {};
       let properties = get(this, 'properties');
       for (let property of properties) {
-        validators[property.key] = validatorsFor(property);
+        // FIXME: probably better to do this in the `schemaParser` but this is quick fix to get schema props to validate when they are inside a section
```

Comment:
The SchemaFormParser only converts the JSON:Schema-form payload into a data structure that can be easily consumed by the component. It is scalar values. The validations however, were meant to be parsed in the component and grafted on. So in short what you are currently doing is probably correct. One could argue that the validations could be a second parser object but I think that is too much abstraction early on. Doing it here seems good enough for now.